maven-archetype-quickstart/src/test/resources/projects/it-java-7/archetype.properties

```diff
@@ -2,4 +2,4 @@ groupId=archetype.it
 artifactId=java-8-quickstart
 version=0.1-SNAPSHOT
 package=it.pkg
-javaCompilerVersion=1.7
+javaCompilerVersion=7
```

Comment:
are you sure "7" will be recognized?

Reply:
you can build and see it works
but this is a good question that I had myself previously: what precisely makes this work? I summarised my findings in https://issues.apache.org/jira/browse/MPOM-449

Reply:
Thanx!